client/src/__tests__/unit-tests/bitbake-driver.test.ts

```diff
@@ -9,13 +9,16 @@ import { BitbakeDriver } from '../../driver/BitbakeDriver'
 
 jest.mock('vscode')
 
-const mockBitbakeConfiguration = (values: Record<string, string>): void => {
+const mockBitbakeConfiguration = (values: Record<string, unknown>): void => {
   vscode.workspace.getConfiguration = jest.fn().mockImplementation(() => ({
-    get: (key: string): string | undefined => {
+    get: (key: string): unknown => {
       if (key === undefined || values[key] === undefined) {
         return ''
       }
       return values[key]
+    },
+    update: async (key: string, value: unknown): Promise<void> => {
```

Comment:
Are you planning to use it in the test?  However, I am not sure if it makes sense to test it since it always returns a `resolve()` no matter how you change the code. Maybe it makes more sense to test it in an integration test.

Reply:
This is needed for here: https://github.com/savoirfairelinux/vscode-bitbake/blob/738f0beef7fd7a2f7f8235ab33bdc245b636b486/client/src/driver/BitbakeSettings.ts#L54

Otherwise `update` is undefined and the tests fail